frontend/packages/dev-console/src/components/topology/filters/filter-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { RootState } from '@console/internal/redux';
+
+export enum ShowFiltersKeyValue {
+  podCount = 'Pod Count',
+  eventSources = 'Event Sources',
+}
+
+export enum ExpandFiltersKeyValue {
+  knativeServices = 'Knative Services',
+  appGrouping = 'Application Groupings',
+  operatorGrouping = 'Operator Groupings',
+}
+
+export const getTopologyFilters = ({
+  plugins: {
+    devconsole: { topology },
+  },
+}: RootState): TopologyFilters => topology.get('filters');
+
+export type TopologyFilters = {
+  display: DisplayFilters;
+  searchQuery: SearchQuery;
+};
```

Comment:
Probably shouldn't create types after they're used... but I suspect TypeScript is handling this. You cannot do this with variables, our linter errors out on it.